src/hypergraph.jl

```diff
@@ -295,4 +295,39 @@ function nhv(h::Hypergraph{T, V, E}) where {T <: Real, V, E}
     length(h.v2he)
 end
 
+"""
+    _walk!(h::Hypergraph, s::Vector{Int}, i::Vector{Int}, visited::Vector{Bool})
+
+Return the list of neighbors of a given vertex (an auxiliary function for get_connected_components).
+"""
+function _walk!(h::Hypergraph, s::Vector{Int}, i::Vector{Int}, visited::Vector{Bool})
```

Comment:
This will not work correctly as `falses(::Int)` returns `BitArray`. Here is a fixed signature. Also `i` is simply an `Int`
```suggestion
function _walk!(h::Hypergraph, s::AbstractVector{Int}, i::Int, visited::AbstractVector{Bool})
```